index.html

```diff
@@ -9,12 +9,12 @@
     <div class="content">
         <header>
             <h1>{{ page.title }} </h1>
-            <p>A free and fully responsive site template</p>
+            <p>Free and community-sourced recipes.</p>
```

Comment:
Fixed!

Reply:
Isn't that redundant with the paragraph directly below the header now?